static/app/components/forms/form.tsx

```diff
@@ -30,6 +30,7 @@ export interface FormProps
     | 'onFieldChange'
     | 'onSubmitError'
     | 'onSubmitSuccess'
+    | 'mapFormErrors'
```

Comment:
This was already on the form model and is used in a couple places, but not on the Form component itself